peripherals/sensors/mlx90397/package.json

```
@@ -0,0 +1,34 @@
+{
+  "name": "mlx90397",
+  "description": "The MLX90397 is the newest addition to the Melexis position sensing portfolio, bringing the highest flexibility in the portfolio’s smallest assembly. Complementing this, the magnetic fi eld sensor is designed for micropower applications, with programmable duty cycles in the range of 0.1% to 100%.",
+  "description_zh": "MLX90397 是来自迈来芯公司的一颗低功耗的3D磁位置传感器",  
+  "enable": "PKG_USING_MLX90397", 
+  "keywords": [
+    "mlx90397"
+  ],
+  "category": "peripherals/sensors",
+  "author": {
+    "name": "lgnq",
+    "email": "dzzxzz@gmail.com",
+    "github": "lgnq"
+  },
+  "license": "Apache-2.0",
+  "repository": "https://github.com/lgnq/mlx90397",
+  "icon": "unknown",
+  "homepage": "https://github.com/lgnq/mlx90397`",
+  "doc": "unknown",
+  "site": [
+    {
+      "version": "v1.0.0",
+      "URL": "https://github.com/lgnq/mlx90397.git",
+      "filename": "mlx90397-1.0.0.zip",
+      "VER_SHA": "1da939f44ab90cab9b2119d65b11139c91808741"
```

发布release版本的时候sha值不需要填写，发布最新版本的时候sha值填写分支名称（听起来有些古怪）
```suggestion
      "VER_SHA": ""
```